tests/conftest.py

```diff
@@ -75,6 +75,7 @@ def pytest_configure():
             'rest_framework_gis',
             'rest_framework.authtoken',
             'crispy_forms',
+            'stored_messages',
```

Comment:
> Are we relying in tests on this app being present? I couldn't find it in the changed code [...]

That would be [the call to](https://github.com/geometalab/osmaxx-frontend/pull/446/files#diff-e52c21d719a322835dc4db89ea595076R58) `osmaxx.utilities.shortcuts.Emissary.info` which uses `stored_messages` [via `Emissary.inform`](https://github.com/geometalab/osmaxx-frontend/blob/0704004e8da31444f947bec022ef60132284224d/osmaxx/utilities/shortcuts.py#L80).